test/code/providers/Container_ImageInventory_Class_Provider_UnitTest.cpp

```diff
@@ -93,30 +105,62 @@ class ImageInventoryTest : public CppUnit::TestFixture
 		StandardTestEnumerateInstances<mi::Container_ImageInventory_Class_Provider>(m_keyNames, context, CALL_LOCATION(errMsg));
 
 		// Get images using command line
-		CPPUNIT_ASSERT(!system("docker images -q --no-trunc > /tmp/docker_image_ids.txt"));
+		CPPUNIT_ASSERT(!system("docker images --no-trunc > /tmp/docker_image_ids.txt"));
 
 		FILE* idFile = fopen("/tmp/docker_image_ids.txt", "r");
 		CPPUNIT_ASSERT(idFile);
 
-		wchar_t id[65];
+		wchar_t line[200];
 		set<wstring> allIds;
 
-		// Full image IDs (one per line)
-		while (fwscanf(idFile, L"%ls", id) != EOF)
+		int matched = 0;
+
+		// Full image info (header plus one per line)
+		if (fgetws(line, 200, idFile) != NULL)
 		{
-			allIds.insert(wstring(id));
+			// parse header to get column offsets
+			wstring hdrstr(line);
+			string::size_type repoix = hdrstr.find(L"REPOSITORY", 0);
+			CPPUNIT_ASSERT(repoix != wstring::npos);
```

Comment:
This type of ASSERT gives very little information if it fires. Can you at least use CPPUNIT_ASSERT_MESSAGE to get a meaningful message of the problem if the unit test fails? Same for the lines below ...

On failure, you should print the line itself plus the fact that you couldn't find "REPOSITORY" within the line. Perhaps something like:

Can't find "REPOSITORY" keyword in docker output line: <docker output line>